must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -191,9 +191,21 @@ for ns in $namespaces; do
         done
     done
 
-    # Add Ready nodes to the list
-    nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
+    #collecting journal logs
+    for node in "$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '{print $1}')"; do
```

Comment:
Why are you doing this instead of looping through `nodes`? Do we really want to loop through non-Ready nodes? If not, can't you just insert this in the other for loop below?

Reply:
This is still a WIP(notice the CI is on hold)  PR and these codes I pushed 5 days ago when I wanted @crombus  to take a look at. I will update correct code once I have completed it locally.